content/rs/references/rladmin.md

```diff
@@ -1,30 +1,697 @@
 ---
```

Comment:
General comment - @AlonMagrafta @kaitlynmichaelredislabs thoughts about how we indicate which commands are `dangerous` - could lead to data loss or `caution` - can have other adverse impacts - if at all?
Some of these are required for setup and regular maintenance while some are advanced and shouldn't be used without consulting Redis Labs support.

Reply:
I think we should mention any potential danger of data loss. I just need to know which commands are "dangerous" and how data loss or any other potential pitfalls can be avoided.

Reply:
It's a good idea. We will need a thorough review of it to be able to mark them all

Reply:
I created DOC-618 for marking the "dangerous" commands